packages/features/shell/Shell.tsx

```diff
@@ -957,7 +957,10 @@ function SideBar({ bannersHeight, user, isPlatformUser = false }: SideBarProps)
     <div className="relative">
       <aside
         style={!isPlatformUser ? sidebarStylingAttributes : {}}
-        className="bg-muted border-muted fixed left-0 hidden h-full max-h-screen w-14 flex-col overflow-y-auto overflow-x-hidden border-r md:sticky md:flex lg:w-56 lg:px-3">
+        className={classNames(
+          "bg-muted border-muted fixed left-0 hidden h-full w-14 flex-col overflow-y-auto overflow-x-hidden border-r md:sticky md:flex lg:w-56 lg:px-3",
+          !isPlatformUser && "max-h-screen"
```

Comment:
I think Shell should not be aware of isPlatformUser, we should refactor to pass props directly to style as needed